Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Install dependencies only when needed
+FROM node:16-alpine AS deps
+ARG NODE_ENV
+RUN apk add --no-cache g++ make py3-pip
+# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
+RUN apk add --no-cache libc6-compat
+WORKDIR /app
+
+# Install dependencies based on the preferred package manager
+COPY ./package*.json ./
+
+
+RUN npm i --legacy-peer-deps  
+
+# Rebuild the source code only when needed
+FROM node:16-alpine AS builder
+
+ARG NEXT_PUBLIC_API_URL
+ARG NEXT_PUBLIC_MEDIA_STORE_URL
+ARG NEXT_PUBLIC_GOOGLE_TAG_MANAGER_ID
+ARG NODE_ENV
```

Comment:
Do we need this variable. As the ENV has the values we need, we don't need to depend upon the NODE_ENV anymore to identify the build type

Reply:
yes we can remove